packages/app/screens/swipe-list.tsx

```diff
@@ -85,6 +89,10 @@ const ProfileSwipeList = () => {
   );
   const { bottom: safeAreaBottom } = useSafeAreaInsets();
 
+  const initialScrollIndex = data?.findIndex(
+    (item) => item.nft_id == initialScrollItemId
+  );
+
```

Comment:
this should useMemo

Reply:
And they might be circumstances, where findIndex return -1, which should handled gracefully with 0